### PR TITLE
changelog1.0.5

### PR DESCRIPTION
**Pull request by dpeng817**

### CODE REVIEW

**Comment by dpeng817** [comment]
Current dependencies on/for this PR:
* master
  * **PR #9491** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/9491" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/9491?utm_source=stack-comment).

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+### New
+
+* [dagit] nbconvert is now installed as an extra in Dagit.
+* Multiple assets can be monitored for materialization using the `multi_asset_sensor` (experimental).
+* Run status sensors can now monitor jobs in external repositories.
+* The `config` argument of `define_asset_job` now works if the job contains partitioned assets.
+* When configuring sqlite-based storages in dagster.yaml, you can now point to environment variables.
+* When emitting `RunRequests` from sensors, you can now optionally supply an `asset_selection` argument, which accepts a list of `AssetKey`s to materialize from the larger job.
+* [dagster-dbt] `load_assets_from_dbt_project` and `load_assets_from_dbt_manifest` now support the `exclude` parameter, allowing you to more precisely which resources to load from your dbt project (thanks @flvndh!)
+
```
Comment (johannkm) [inline CHANGES.md:14]:
```suggestion
* [dagster-k8s] `schedulerName` is now available for all deployments in the Helm chart for users who use a custom Kubernetes scheduler
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+### New
+
+* [dagit] nbconvert is now installed as an extra in Dagit.
+* Multiple assets can be monitored for materialization using the `multi_asset_sensor` (experimental).
+* Run status sensors can now monitor jobs in external repositories.
+* The `config` argument of `define_asset_job` now works if the job contains partitioned assets.
+* When configuring sqlite-based storages in dagster.yaml, you can now point to environment variables.
+* When emitting `RunRequests` from sensors, you can now optionally supply an `asset_selection` argument, which accepts a list of `AssetKey`s to materialize from the larger job.
+* [dagster-dbt] `load_assets_from_dbt_project` and `load_assets_from_dbt_manifest` now support the `exclude` parameter, allowing you to more precisely which resources to load from your dbt project (thanks @flvndh!)
+* [dagster-k8s] `schedulerName` is now available for all deployments in the Helm chart for users who use a custom Kubernetes scheduler
```
Comment (dpeng817) [inline CHANGES.md:14]:
```suggestion
* [dagster-k8s] `schedulerName` is now available for all deployments in the Helm chart for users who use a custom Kubernetes scheduler

```

**Review by dpeng817 — COMMENTED** [review]
@johannkm mind stamping?

**Review by sryza — APPROVED** [review]

**Pull request merged**